src/components/skills-quiz/SearchCourseCard.jsx

```diff
@@ -32,11 +36,33 @@ const linkToCourse = (course, slug) => {
   return `/${slug}/course/${course.key}?${qs.stringify(queryParams)}`;
 };
 
+const renderDialog = () => (
+  <div className="lead d-flex align-items-center py-3">
+    <div className="mr-3">
+      <FontAwesomeIcon icon={faSearchMinus} size="2x" />
+    </div>
+    <p>
+      { NO_COURSES_ALERT_MESSAGE }
+    </p>
+  </div>
+);
+
 const SearchCourseCard = ({ index }) => {
-  const { enterpriseConfig: { slug } } = useContext(AppContext);
+  const { enterpriseConfig } = useContext(AppContext);
+  const { slug } = enterpriseConfig;
+  const { subscriptionPlan, subscriptionLicense, offers: { offers } } = useContext(UserSubsidyContext);
+  const offerCatalogs = offers.map((offer) => offer.catalog);
+  const { filters } = useDefaultSearchFilters({
+    enterpriseConfig,
+    subscriptionPlan,
+    subscriptionLicense,
+    offerCatalogs,
+  });
+
   const { state } = useContext(SkillsContext);
   const [isLoading, setIsLoading] = useState(true);
   const [courses, setCourses] = useState([]);
+  const [hitCount, setHitCount] = useState(undefined);
```

Comment:
Seems like `0` would be a better default for `hitCount`.

Reply:
I have intentionally set this to `undefined`. In case if no courses are returned from Algolia, only then the HitCount should be 0. I am rendering a StatusAlert based on this condition.